netlify/functions/submit-finding.ts

```diff
@@ -248,35 +252,54 @@ exports.handler = async (event) => {
   }
 
   const owner = process.env.GITHUB_CONTEST_REPO_OWNER;
-  const riskCode = getRiskCodeFromGithubLabel(risk);
 
   try {
-    const markdownPath = `data/${attributedTo}-${riskCode}.md`;
-    const qaOrGasSubmissionBody = `See the markdown file with the details of this report [here](https://github.com/${owner}/${repo}/blob/main/${markdownPath}).`;
-    const isQaOrGasSubmission = Boolean(riskCode === "G" || riskCode === "Q");
-    if (isQaOrGasSubmission) {
-      const existingReport = await getMarkdownReportForUser(
-        octokit,
-        repo,
-        attributedTo,
-        riskCode as "Q" | "G"
-      );
-      if (existingReport) {
-        throw {
-          status: 400,
-          message: `It looks like you've already submitted a ${risk} report for this contest.`,
-        };
+    let isQaOrGasSubmission = false;
+    let qaOrGasSubmissionBody = '';
+    let markdownPath = '';
+    let riskCode = '';
+    if(!isMitigationReport){
+      const riskCode = getRiskCodeFromGithubLabel(risk);
+      markdownPath = `data/${attributedTo}-${riskCode}.md`;
+      qaOrGasSubmissionBody = `See the markdown file with the details of this report [here](https://github.com/${owner}/${repo}/blob/main/${markdownPath}).`;
+      isQaOrGasSubmission = Boolean(riskCode === "G" || riskCode === "Q");
+      if (isQaOrGasSubmission) {
+        const existingReport = await getMarkdownReportForUser(
+          octokit,
+          repo,
+          attributedTo,
+          riskCode as "Q" | "G"
+        );
+        if (existingReport) {
+          throw {
+            status: 400,
+            message: `It looks like you've already submitted a ${risk} report for this contest.`,
+          };
+        }
       }
     }
 
+    //if it's a confirmed mitigation then no title will be provided
+    let isConfirmedMitigation = false;
+    if(isMitigationReport && (
+      !body || !title || !risk
+    )){
+      isConfirmedMitigation = true;
+    }else if(
+      isMitigationReport
+    ){
+      labels.push(`bug`)
+      labels.push(`MR-${mitigationOf}`)
+    }
+
     const issueResult = await octokit.request(
       "POST /repos/{owner}/{repo}/issues",
       {
         owner,
         repo,
-        title,
+        title : !isConfirmedMitigation? title : mitigationTitle,
```

Comment:
if you follow one of my first suggestions for defining mitigationTitle, you can do this:

```javascript
title: mitigationTitle || title,
```